code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -97,6 +97,71 @@
 	name = "trophy rack"
 	desc = "It's useful for both carrying extra gear and proudly declaring your insanity."
 	icon_state = "cultpack"
+	var/obj/item/organ/external/head/LS
+	var/obj/item/organ/external/head/MS
+	var/obj/item/organ/external/head/RS
+	var/mutable_appearance/LSma
+	var/mutable_appearance/MSma
+	var/mutable_appearance/RSma
+	var/list/placement
```

Comment:
Why is this a list here? It's used as a string on line 111.

Reply:
Fixed